Airports/EGCC/SMR/Geo.txt

```diff
@@ -1243,15 +1243,25 @@ N053.21.59.434 W002.17.20.195 N053.21.59.748 W002.17.20.357 smrCCtaxiway
 N053.21.59.748 W002.17.20.357 N053.21.59.983 W002.17.20.635 smrCCtaxiway
 N053.21.59.983 W002.17.20.635 N053.22.02.593 W002.17.25.058 smrCCtaxiway
 N053.22.02.593 W002.17.25.058 N053.22.02.750 W002.17.25.303 smrCCtaxiway
-N053.22.02.750 W002.17.25.303 N053.22.02.957 W002.17.25.491 smrCCtaxiway
-N053.22.02.957 W002.17.25.491 N053.22.03.181 W002.17.25.592 smrCCtaxiway
-N053.22.03.181 W002.17.25.592 N053.22.03.395 W002.17.25.606 smrCCtaxiway
-N053.22.03.395 W002.17.25.606 N053.22.03.584 W002.17.25.549 smrCCtaxiway
-N053.22.03.584 W002.17.25.549 N053.22.03.817 W002.17.25.419 smrCCtaxiway
-N053.22.03.817 W002.17.25.419 N053.22.05.506 W002.17.24.415 smrCCtaxiway
-N053.22.05.506 W002.17.24.415 N053.22.05.776 W002.17.24.181 smrCCtaxiway
-N053.22.05.776 W002.17.24.181 N053.22.05.977 W002.17.23.889 smrCCtaxiway
-N053.22.05.977 W002.17.23.889 N053.22.13.000 W002.17.12.236 smrCCtaxiway
+N053.22.02.750 W002.17.25.303 N053.22.02.957 W002.17.25.491 smrCCOrange
+N053.22.02.957 W002.17.25.491 N053.22.03.181 W002.17.25.592 smrCCOrange
+N053.22.03.181 W002.17.25.592 N053.22.03.395 W002.17.25.606 smrCCOrange
+N053.22.03.395 W002.17.25.606 N053.22.03.584 W002.17.25.549 smrCCOrange
+N053.22.03.584 W002.17.25.549 N053.22.03.817 W002.17.25.419 smrCCOrange
+N053.22.03.817 W002.17.25.419 N053.22.05.506 W002.17.24.415 smrCCOrange
+N053.22.05.506 W002.17.24.415 N053.22.05.776 W002.17.24.181 smrCCOrange
+N053.22.05.776 W002.17.24.181 N053.22.05.977 W002.17.23.889 smrCCOrange
+N053.22.05.977 W002.17.23.889 N053.22.13.000 W002.17.12.236 smrCCOrange
+
+;Taxiway_Z_Blue
```

Comment:
I would have left this in its alphabetical position within the file, makes finding them easier in future.

Reply:
I'd would have preferred them to be together my opinion

Reply:
So now you've separated the Z Orange off into a separate section, you can easily have Z Blue and Z Orange both together and in alphabetical order in the file - i.e. Pop the Z Blue section back where it was and place Z Orange after it. It might also be a good idea to adjust the title of the `;Taxiways_E,_Z_and_Z_Orange` section to omit the Z Orange bit now it's separate :) Thanks @rishab-alt

Reply:
Okay , will do that

Reply:
> Okay , will do that

Sorry I clearly confused you as you haven't done what I was meaning... you've re-clumped them into one section! I'll just edit the file directly to show you what I mean, as it's not going to let me do it as a suggestion easily I don't think, sorry again for the confusion!